spec/factories/users.rb

```diff
@@ -2,14 +2,9 @@
   factory :user do
     password { 'password' }
 
-    factory :user_with_email_phone do
-      email { 'user@example.com' }
-      phone { '123-456-7890' }
-    end
-
-    factory :correct_user_update do
-      email { 'user@example.com' }
-      phone { '123-456-7890' }
+    trait :correct_user_update do
+      email { Faker::Internet.email }
+      phone { Faker::Base.numerify('###-###-####') }
```

Comment:
is that valid phone number?
https://github.com/faker-ruby/faker/blob/master/doc/default/phone_number.md

Reply:
Not at the moment because earlier in the model the following expression was used / \ (? [0-9] {3} \)? - [0-9] {3} - [0-9] {4} /
And because faker generates in a random format, not only through the hyphen, it is not always valid, for its use then it is worth changing the regular expression in the validation of the phone number.